graphics/nanovg/Portfile

```diff
@@ -11,10 +11,11 @@ maintainers         @jasonliu-- openmaintainer
 
 if {$subport eq "nanovg"} {
 
-github.setup        memononen $subport 2bead03bea43b2418060aaa154f972829995e663
-version             20200320
-revision            0
-
+set shorthash       bf2320d
+set fullhash        bf2320d1175122374a9b806d91e9e666c9336375
+set lastcommit      20190524
```

Comment:
please avoid setting these local variables, they are unnecessary and in fact @ryandesign removed them in his cleanup commits

https://github.com/macports/macports-ports/commits/master/graphics/nanovg

Please follow the same principle as in the current master version.

Reply:
Gotcha. Sorry, I hadn't noticed those changes... I basically just took a large chunk from my file and replaced what was in master wholesale. I'll take a closer look and make sure that my updates retain what's in master.

Reply:
OK. its generally a good idea to keep you checkout up to date (i.e. rebased) against master regularly.